tests/setup-jest.js

```diff
@@ -4,3 +4,8 @@ jest.mock('src/services/google-analytics');
 process.on('unhandledRejection', (reason, promise) => {
   console.log('unhandledRejection', reason, promise);
 });
+
+const { TextEncoder, TextDecoder } = require('util'); // eslint-disable-line @typescript-eslint/no-var-requires
```

Comment:
Since this is an import, could you please move it to the top of the file?

Reply:
✅